mappings/net/minecraft/client/option/GameOptions.mapping

```diff
@@ -45,50 +75,78 @@ CLASS net/minecraft/class_315 net/minecraft/client/option/GameOptions
+		COMMENT A key binding for holding an item corresponding to the entity or block the player is looking at
+		COMMENT Bound to {@linkplain org.lwjgl.glfw.GLFW#GLFW_MOUSE_BUTTON_MIDDLE the middle mouse button} by default.
 	FIELD field_18725 backgroundForChatOnly Z
 	FIELD field_18726 textBackgroundOpacity D
 	FIELD field_1872 overrideWidth I
 	FIELD field_1873 autoSuggestions Z
 	FIELD field_1874 keyLoadToolbarActivator Lnet/minecraft/class_304;
+		COMMENT A key binding for loading the hotbar items in {@linkplain net.minecraft.world.GameMode#CREATIVE creative mode}.
+		COMMENT Bound to {@linkplain org.lwjgl.glfw.GLFW#GLFW_KEY_X the X key} by default.
 	FIELD field_1875 tutorialStep Lnet/minecraft/class_1157;
 	FIELD field_1876 useNativeTransport Z
 	FIELD field_1877 chatVisibility Lnet/minecraft/class_1659;
 	FIELD field_1878 biomeBlendRadius I
 	FIELD field_1879 keySaveToolbarActivator Lnet/minecraft/class_304;
+		COMMENT A key binding for saving the hotbar items in {@linkplain net.minecraft.world.GameMode#CREATIVE creative mode}.
+		COMMENT Bound to {@linkplain org.lwjgl.glfw.GLFW#GLFW_KEY_C the C key} by default.
 	FIELD field_1880 debugProfilerEnabled Z
 	FIELD field_1881 keyBack Lnet/minecraft/class_304;
+		COMMENT A key binding for moving backward.
+		COMMENT Bound to {@linkplain org.lwjgl.glfw.GLFW#GLFW_KEY_S the S key} by default.
 	FIELD field_1882 particles Lnet/minecraft/class_4066;
 	FIELD field_1883 language Ljava/lang/String;
 	FIELD field_1884 enableVsync Z
 	FIELD field_1885 overrideHeight I
 	FIELD field_1886 keyAttack Lnet/minecraft/class_304;
+		COMMENT A key binding for attacking an entity or breaking a block.
+		COMMENT Bound to {@linkplain org.lwjgl.glfw.GLFW#GLFW_MOUSE_BUTTON_LEFT the left mouse button} by default.
 	FIELD field_1887 resourcePacks Ljava/util/List;
 	FIELD field_1888 entityShadows Z
 	FIELD field_1889 mouseWheelSensitivity D
 	FIELD field_1890 keyChat Lnet/minecraft/class_304;
+		COMMENT A key binding for opening {@linkplain net.minecraft.client.gui.screen.ChatScreen the chat screen}.
+		COMMENT Bound to {@linkplain org.lwjgl.glfw.GLFW#GLFW_KEY_T the T key} by default.
 	FIELD field_1891 bobView Z
 	FIELD field_1892 enabledPlayerModelParts Ljava/util/Set;
 	FIELD field_1893 debugTpsEnabled Z
 	FIELD field_1894 keyForward Lnet/minecraft/class_304;
+		COMMENT A key binding for moving forward.
+		COMMENT Bound to {@linkplain org.lwjgl.glfw.GLFW#GLFW_KEY_W the W key} by default.
 	FIELD field_1895 attackIndicator Lnet/minecraft/class_4061;
 	FIELD field_1896 narrator Lnet/minecraft/class_4065;
 	FIELD field_1897 optionsFile Ljava/io/File;
 	FIELD field_1900 chatColors Z
 	FIELD field_1901 glDebugVerbosity I
 	FIELD field_1903 keyJump Lnet/minecraft/class_304;
+		COMMENT A key binding for jumping.
+		COMMENT Bound to {@linkplain org.lwjgl.glfw.GLFW#GLFW_KEY_SPACE the space key} by default.
 	FIELD field_1904 keyUse Lnet/minecraft/class_304;
+		COMMENT A key binding for using an item or placing a block.
```

Comment:
"placing a block" is actually one of the instances of "using an item", so I suggest

```suggestion
		COMMENT A key binding for using an item, such as placing a block.
```